hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/conf/OzoneManagersCommandHandler.java

```diff
@@ -28,7 +28,7 @@
  * Handler for ozone getconf ozonemanagers.
  */
 @Command(name = "ozonemanagers",
-    aliases = {"-ozonemanagers"},
+    aliases = {"-ozonemanagers", "--ozone-managers"},
```

Comment:
`-ozonemanagers` here is not an option, rather an alias for the command name (`ozonemanagers`).  I think we should omit the new alias.

```suggestion
    aliases = {"-ozonemanagers"},
```